.github/workflows/linux-ci.yml

```diff
@@ -31,3 +33,45 @@ jobs:
           cargo test --package libparsec_crypto --features use-rustcrypto
 
           cargo test --workspace --features mock-time --exclude libparsec_crypto
+
+  python:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # pin v3.0.2
+
+      - uses: ./.github/actions/setup-rust
+        with:
+          version: ${{ env.rust-version }}
+          cache-key: python
+
+      - name: Install poetry
+        env:
+          POETRY_VERSION: ${{ env.poetry-version }}
```

Comment:
I'd rather use `... | python - --version=$POETRY_VERSION` (or `... --version=${{ env.poetry-version }}`)  instead of directly the env version, this is to be more explicit and be sure the environ variable doesn't at one point get ignored due to an unexpected changed in the get-poetry script

Reply:
done